File: Listas antigas em C/Lista 1/Questão 10 Lista 1.c
/*
Questão 10 Lista 1
Faça um algoritmo para ler 3 números inteiros, depois calcular e imprimir a média aritmética destes.
*/
#include <stdio.h>  

int main(void) {
    int a, b, c;
    float media;

	printf("digite a, b e c\n");
	scanf("%d",&a);
	scanf("%d",&b);
	scanf("%d",&c);
	media = (a+b+c)/3;
	printf("media = %f", media);
    return 0; 
}

File: Lista 1/Questão 10 Lista 1.py
"""
Questão 10 Lista 1
Faça um algoritmo para ler 3 números inteiros, depois calcular e imprimir a média aritmética destes.
"""
numeros=[]
for numero in range(1, 4):
	numeros.append(int(input(f"{numero}º número: ")))
print(f"\nMédia = {sum(numeros)/len(numeros)}")
